src/components/Navbar.tsx

```diff
@@ -12,7 +13,14 @@ function NavItem(props: { name: string; children: PropTypes.ReactNodeLike }) {
   );
 }
 
+
 function Navbar(): JSX.Element {
+  const [searchItem, setSearchItem] = useState("");
+
+  function handleChange(e: any) {
```

Comment:
Do we really need this function that just forwards it's arguments to another function?